rustfmt.toml

```
@@ -0,0 +1,3 @@
+comment_width = 140
```

This should be changed to 100 to match bitcoindevkit/bdk#1946 then this one should be good to go in.
```suggestion
comment_width = 100
```

---

And it also needs a rebase to pickup latest CI changes.

---

Done.